SECURITY_CONTACTS

```diff
@@ -1,10 +1,10 @@
 # Defined below are the security contacts for this repo.
 #
-# They are the contact point for the Product Security Team to reach out
```

Comment:
FYI this is synced from https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apimachinery.

Reply:
I couldn't remember if this was a one-off or from staging. Thanks for clarifying. We can close this?

Reply:
I believe so.